Dockerfile

```diff
@@ -23,9 +23,13 @@ RUN set -ex; \
   mv /usr/local/src/courier/courier /usr/local/bin/; \
   rm -rf /usr/local/src/courier courier.tar.gz; \
   \
-  apt-get purge -y --auto-remove wget
+  apt-get purge -y --auto-remove wget;\
+  addgroup --system courier; \
```

Comment:
I would create the user and group at the start of the Dockerfile since they basically never change.